src/EPR.Payment.Mopup/PaymentsFunction.cs

```diff
@@ -24,14 +19,13 @@ public PaymentsFunction(IPaymentsService paymentsService, ILogger<PaymentsFuncti
             _logger = logger;
         }
 
-        [FunctionName("PaymentsFunction")]
-        public async Task Run([Sql("Select * from Payment where InternalStatusId = 1", commandType: CommandType.Text, connectionStringSetting: "SqlConnectionString")] IEnumerable<Common.Dtos.Request.Payment> payments,
-                        [Sql("Payment", connectionStringSetting: "SqlConnectionString")] IAsyncCollector<Common.Dtos.Request.Payment> paymentToUpdate,
-                        [TimerTrigger("%FUNCTIONS_TIME_TRIGGER%")] TimerInfo myTimer, CancellationToken cancellationToken)
+        [FunctionName("MopUpPaymentsFunction")]
```

Comment:
Better suits `MopUpPaymentsTimerTriggerFunction` ?

Reply:
We both decided to keep the same